### ISSUE TITLE
[BITV]: Replace custom tooltips with native one for Mobile & desktop settings page (firstrunwizard app)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#35906]
## Problem

To show a tooltip on a several components we currently use v-tooltip. An example can be found in https://github.com/nextcloud/firstrunwizard/blob/ae5062447c1a9194f35abbdd7e6dd324ac3b3dba/templates/personal-settings.php#L96

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to several components that will render as `title` attribute on the HTML element.

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/firstrunwizard/issues/804
